### DISCUSSION TITLE
Command Palette?

### DISCUSSION THREAD

**Discussion by rsyring** [dbeaver/dbeaver#35599] [General]
DBeaver has extensive keyboard combinations.  However, remembering them all, and evening typing some in (shift+alt+ctrl+A), can be a challenge.

I find VS Code's command palette (ctrl+p) ideal in this respect.  Does DBeaver have anything like the command palette, where, in a single dialog, one can search for commands, files, objects all in the same place?

Thanks.

--- 

FFR, I've now found:

* Find actions (ctrl+3)

**Comment by E1izabeth** [comment]
DBeaver also have Key Assist on `Ctlr+Shift+L`
![image](https://github.com/user-attachments/assets/1a938435-db94-4739-b53f-52bd74363713)

**Reply by TB-HunBoka** [reply]
I didn't know that, love it. Thanks.

**Comment by bonniesimon** [comment]
Pressing "Command + 3" is giving me a spotlight like interface.

**Reply by resting** [reply]
For those wondering what's cmd + 3, Find Actions
![CleanShot 2025-06-11 at 2  36 41@2x](https://github.com/user-attachments/assets/f7605e94-3f55-40bd-83bc-abb14b36e89e)
